packages/browsers/src/CLI.ts

```diff
@@ -222,7 +222,31 @@ export class CLI {
           );
           yargs.example(
             '$0 install firefox',
-            'Install the latest available build of the Firefox browser.'
+            'Install the latest nightly available build of the Firefox browser.'
+          );
+          yargs.example(
+            '$0 install firefox@stable',
+            'Install the latest stable build of the Firefox browser.'
+          );
+          yargs.example(
+            '$0 install firefox@beta',
+            'Install the latest beta build of the Firefox browser.'
+          );
+          yargs.example(
+            '$0 install firefox@devedition',
+            'Install the latest devedition build of the Firefox browser.'
+          );
+          yargs.example(
+            '$0 install firefox@esr',
+            'Install the latest ESR build of the Firefox browser.'
```

Comment:
Should we initially support ESR releases? You don't do that with Chrome (if you have such ones) and maybe we should align for now. It would certainly cause breakage due to not implemented features. Or will we have a support matrix of which version of puppeteer supports which ESR release?

Reply:
I think we should as this is the `@puppeteer/browsers` package and as long as a distribution of said browser we should have a way to download it. 
This PR does not integrate it with `puppeteer` so users will not see breakage. I think support matix for Firefox will become a must if we don't use `nightly` for downloads from `puppeteer`.

Reply:
Sounds fine then. I only wanted to give the heads-up that it might need further work on Puppeteer's side to exclude specific Puppeteer releases for older Firefox ESR versions.